pkg/kudoctl/cmd/init.go

```diff
@@ -53,11 +61,13 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 			if err := i.validate(); err != nil {
 				return err
 			}
+			i.home = Settings.Home
 			return i.run()
 		},
 	}
 
 	f := cmd.Flags()
+	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "If set does not install KUDO")
```

Comment:
what about calling this `local` ... client is very generic word and even in the code we call client kubernetes client, which made it a bit harder to understood for me